apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -13,6 +23,36 @@ export default class ModerationNotification extends Component {
     )
   }
 
+  async toggleIsPending () {
+    const data = { is_pending: !this.state.isPending }
+    const headers = {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      'X-CSRFToken': Cookies.get('csrftoken')
+    }
+
+    fetch(this.props.apiUrl, {
+      method: 'PATCH',
+      headers: headers,
+      body: JSON.stringify(data)
+    })
+      .then(res => res.json())
+      .then(json => {
+        this.setState({
+          isPending: json.is_pending
+        })
+      }).catch((err) => {
+        console.log(err)
+      })
+  }
+
+  toggleIsBlocked () {
+    const newIsBlocked = !this.state.isBlocked
```

Comment:
Just for my understanding:
this is in reference to the comment variable **is_blocked** comiing from a4, right? And will this be added to the ModerationNotificationList.jsx and changed accordingly, which is a similar way that blocked comments will then not be shown in discussion? (I'm going to try this next and still aiming to grasp the setup)

Reply:
The first part: yes. @phillimorland just added a placeholder which will be filled once the actual field is merged.
And the second part I do not quite get. Here it's a button but for the discussion, we would either not show them or show them differently depending on the value of is_blocked.